zcash_client_sqlite/src/wallet/init/migrations/receiving_key_scopes.rs

```diff
@@ -0,0 +1,111 @@
+//! This migration adds support for the Orchard protocol to `zcash_client_sqlite`
+
+use std::collections::HashSet;
+
+use rusqlite::{self, named_params};
+use schemer;
+use schemer_rusqlite::RusqliteMigration;
+
+use tracing::debug;
+use uuid::Uuid;
+
+use zcash_client_backend::{keys::UnifiedFullViewingKey, scanning::ScanningKey};
+use zcash_primitives::{
+    consensus::{self, sapling_zip212_enforcement, BlockHeight, BranchId},
+    sapling::note_encryption::{try_sapling_note_decryption, PreparedIncomingViewingKey},
+    transaction::Transaction,
+    zip32::Scope,
+};
+
+use crate::wallet::{
+    init::{migrations::shardtree_support, WalletMigrationError},
+    scope_code,
+};
+
+pub(super) const MIGRATION_ID: Uuid = Uuid::from_u128(0xee89ed2b_c1c2_421e_9e98_c1e3e54a7fc2);
+
+pub(super) struct Migration<P> {
+    pub(super) params: P,
+}
+
+impl<P> schemer::Migration for Migration<P> {
+    fn id(&self) -> Uuid {
+        MIGRATION_ID
+    }
+
+    fn dependencies(&self) -> HashSet<Uuid> {
+        [shardtree_support::MIGRATION_ID].into_iter().collect()
+    }
+
+    fn description(&self) -> &'static str {
+        "Add support for receiving storage of note commitment tree data using the `shardtree` crate."
+    }
+}
+
+impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
+    type Error = WalletMigrationError;
+
+    fn up(&self, transaction: &rusqlite::Transaction) -> Result<(), WalletMigrationError> {
+        // Add commitment tree sizes to block metadata.
+        debug!("Adding new columns");
+        transaction.execute_batch(
+            &format!(
+                "ALTER TABLE sapling_received_notes ADD COLUMN recipient_key_scope INTEGER NOT NULL DEFAULT {};",
```

Comment:
```suggestion
                "ALTER TABLE sapling_received_notes ADD COLUMN recipient_key_scope INTEGER NOT NULL DEFAULT 0;",
```
I think that if we specify the default here then it doesn't need to be specified as a property of the column.